Resources/views/Form/fields.html.twig

```diff
@@ -102,4 +102,23 @@
       <span class="input-group-addon"><i style="background: {{ value }}"></i></span>
     </div>
   {% endspaceless %}
-{% endblock color_widget %}
\ No newline at end of file
+{% endblock color_widget %}
+
+{% block select_ajax_widget %}
+  {% block javascript %}
+    {{ parent() }}
+    {% spaceless %}
+      <script src="{{ path('_form.select.ajax.js') }}" type="application/javascript"></script>
```

Comment:
you should indicate in the documentation how to include your javascript libraries, not include the library for each time the widget is used